README.md

```diff
@@ -424,10 +424,13 @@ Tracing.
 
 * [Bay Area Rust Meetup talk and Q&A][bay-rust-2018-03], March 2018
 * [RustConf 2019 talk][rust-conf-2019-08-video] and [slides][rust-conf-2019-08-slides], August 2019
+* Are we observable yet? @ RustyDays - [talk][rusty-days-2020-08-video] and [slides][rusty-days-2020-08-slides], August 2020
```

Comment:
nit: for the other two entries here, i made the title of the talk link to the video. maybe we should do that here, as well, for consistency?

Reply:
Done :heavy_check_mark: